styles/style.css

```
@@ -1,10 +1,75 @@
-/*
-Colors to use:
-dark blue: #06365f
-light blue: #1c5380
-*/
 
 body {
+  background-color: #06365f;
   font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
   margin: 0;
+  padding: 70px 60px 30px;
```

Eduardo, ya solamente te faltó aplicar la imagen que venia con truco, el detalle está en traer la imagen desde css, lo cual dices que se te complica. El detalle esta en que aplicamos una imagen de fondo y la imagen se trae con un tipo de funcion en css que es url la sintaxis es la siguiente: `background-image: url("images/bgtile.png");`.

Y aun viene con otro truco al aplicar el fondo como en la imagen  y es decirle que en el eje Y no se repita, solo en el x, lo que hace el efecto de degradado.
```suggestion
  padding: 70px 60px 30px;
  margin: 0;
  background-image: url("images/bgtile.png");
  background-repeat-y: no-repeat;
```